### PR TITLE
Feature/deploy bridge

### PR DESCRIPTION
**Pull request by iStrike7**
Automated bridge deployment on Kovan

### CODE REVIEW

File: deploy-bridge/.env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SUB_CHAIN_RECEIPIENT=0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d
+SUB_CHAIN_RECEIPIENTSS58=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+ETH_ENDPOINT=https://kovan.infura.io/v3/e8b4790b8e4049cca3c04f738cfa25f2
+SUB_ENDPOINT=ws://parachain:9944/
+ethAddress0=0xa5d3f778d0615977cdd7d12c05aaff34e4b3474b
+ethPrivateKey0=0x8942a0094dfda5c7d40db448588ffb58513802f7c48edaded7bf6157f026567d
+ethAddress1=0x47b9c50c24891bff382004ac73a9e53a230a2802
+ethPrivateKey1=0x62e5c324be3b4a5dc690d66f43f033bb048a3ffb645a8973cb45b436698e27cd
```
Comment (gleb-urvanov) [inline deploy-bridge/.env:8]: why do we need two eth keys?
Also, please remove the .env file from the repo to hide the sensitive information, and cover required env variables in documentation instead.

File: ethereum/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+BRIDGE_DEPLOY_PATH=/work/mangata-ws/mangata-bridge/test
+ETH_ENDPOINT_WS=ws://ganache:8545/
+SUB_ENDPOINT_WS=ws://parachain:9944/
```
Comment (gleb-urvanov) [inline ethereum/.env:3]: `SUB_ENDPOINT` from relay environment have the same value (and should do it always as far as I can see). Can we combine these two variables into one?

File: ethereum/.env.example [hunk 3]
```diff
@@ -6,3 +6,9 @@ MNEMONIC="candy taste cake sugar pudding cream waffles rich smooth crumble sweet
 
 # INFURA_PROJECT_ID is the project's Infura ID
 INFURA_PROJECT_ID=e8b4790b8e4049cca3c04f738cfa25f2
+
+MNEMONIC
+INFURA_PROJECT_ID
+BASE_PATH
+ETH_ENDPOINT
+SUB_ENDPOINT
```
Comment (gleb-urvanov) [inline ethereum/.env.example:14]: There are inconsistency between .env and .env.example:
.example contains `SUB_ENDPOINT` while .env contains `SUB_ENDPOINT_WS`, same for ETH_ENDPOINT.
Some variables from .example are not seen in .env

File: deploy-bridge/README.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# E2E tests
```
Comment (gleb-urvanov) [inline deploy-bridge/README.md:1]: Is this readme related?

File: deploy-bridge/steps-to-deploy-bridge.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	- SUB_DIR - This is the path to the mangata-node folder. This should contain the compiled node.
+	- MGA_Kovan_Bridge_SUB_ENDPOINT - This is the endpoint for the substrate chain and should be consistent with SUB_CHAIN_PORTS and the parachain container's docker label.
+2. Run `bash init.sh` from /deploy-bridge/deploy
+    - This command will setup the env files, deploy contracts, and deploy the docker services.
+	- It will require you to enter the sudo password.
+	- It will ask you to update the mangataSpec.json manually providing the values to be used.
+	- The flow is as follows := init.sh -> deploy-truffle.sh -> exec-truffle-scripts.sh -> update-mangataSpec.sh -> deploy-services.sh
+
+## Testing the deployment
+You can test the bridge deployment by running yarn test. This will use the above setup env.
```
Comment (gleb-urvanov) [inline deploy-bridge/steps-to-deploy-bridge.md:20]: Please add full list of required environment variables

**Review by gleb-urvanov — CHANGES_REQUESTED** [review]

File: deploy-bridge/.env (hunk 1 above)
Comment (iStrike7) [reply deploy-bridge/.env:8]: Two keys are required because the first derived account in used in Tests as a user.

Comment (iStrike7) [reply deploy-bridge/.env:8]: Do you want me to remove all .env files and use exported variables everywhere or just for these two private keys?

File: ethereum/.env (hunk 2 above)
Comment (iStrike7) [reply ethereum/.env:3]: Sure.

File: ethereum/.env.example (hunk 3 above)
Comment (iStrike7) [reply ethereum/.env.example:14]: .env.example is an old file that should be deleted. I will delete it.

File: deploy-bridge/README.md (hunk 4 above)
Comment (iStrike7) [reply deploy-bridge/README.md:1]: I see there are a couple of files that are redundant and old. I will clean that.

File: deploy-bridge/steps-to-deploy-bridge.md (hunk 5 above)
Comment (iStrike7) [reply deploy-bridge/steps-to-deploy-bridge.md:20]: Sure.

File: deploy-bridge/.env (hunk 1 above)
Comment (gleb-urvanov) [reply deploy-bridge/.env:8]: Ideally not have them at all, and force the user to set all the variables for the deployment. Obviously, examples are fine. Thus probably the best way to move on is to have .env.example in the repo, and .env excluded in gitignore

**Review by gleb-urvanov — APPROVED** [review]
